src/platform/silabs/CHIPDevicePlatformConfig.h

```diff
@@ -61,6 +61,11 @@
 #define CHIP_DEVICE_CONFIG_ENABLE_IPV4 0
 #endif /* CHIP_DEVICE_CONFIG_ENABLE_IPV4 */
 
+#ifdef CHIP_DEVICE_CONFIG_ENABLE_SED
+#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL chip::System::Clock::Milliseconds32(300)
```

Comment:
Are these defines actually used for the WiFi implementation

Reply:
Actually these could and should be removed. I believe they are already present in the default configuration of Matter. Since we are using Hardcoded value anyways, no need to add another defines.

Reply:
True but the values given there is 
```
#ifndef CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL
#define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL 5000_ms32
#endif
```
therefor using the same define and using the DTIM interval active and idle

Reply:
> Are these defines actually used for the WiFi implementation

There are used in `_GetSEDIntervalConfig ` to get the interval configuration for DTIM so we are using these hardcoded values rather than 5000ms and 200ms provided in matter stack

Reply:
perfect thanks.